content/master/concepts/patch-and-transform.md

```diff
@@ -0,0 +1,1692 @@
+
+The 
+{{<hover label="toComposite" line="12">}}ToCompositeFieldPath{{</hover>}} writes 
+data from an individual managed resource to
+the composite resource that created it.
+
+{{< hint "tip" >}}
+Use {{<hover label="toComposite" line="12">}}ToCompositeFieldPath{{</hover>}}
+patches to take data from one managed resource in a Composition and use it in a
+second managed resource in the same Composition. 
+{{< /hint >}}
+
+For example, after Crossplane creates a new managed resource, take the value 
+{{<hover label="toComposite" line="13">}}hostedZoneID{{</hover>}} and apply it
+as a 
+{{<hover label="toComposite" line="14">}}label{{</hover>}} in the composite
+resource.
+
+```yaml {label="toComposite",copy-lines="9-11"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+# Removed for brevity
+    - name: bucket1
+      base:
+        apiVersion: s3.aws.upbound.io/v1beta1
+        kind: Bucket
+        spec:
+          forProvider:
+            region: us-east-2
+      patches:
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.atProvider.hostedZoneId
+          toFieldPath: metadata.labels['ZoneID']
+```
+
+View the created managed resource to see the 
+{{<hover label="toCompMR" line="6">}}Hosted Zone Id{{</hover>}} field.
+```yaml {label="toCompMR",copy-lines="none"}
+$ kubectl describe bucket
+Name:         my-example-claim-p5pxf-5vnp8
+# Removed for brevity
+Status:
+  At Provider:
+    Hosted Zone Id:       Z2O1EMRO9K5GLX
+    # Removed for brevity
+```
+
+Next view the composite resource and confirm the patch applied the 
+{{<hover label="toCompositeXR" line="3">}}label{{</hover>}}
+```yaml {label="toCompositeXR",copy-lines="none"}
+$ kubectl describe composite
+Name:         my-example-claim-p5pxf
+Labels:       ZoneID=Z2O1EMRO9K5GLX
+```
+
+{{<hint "important">}}
+Crossplane doesn't apply the patch to the composite resource until the next
+reconcile loop, after creating the managed resource. This creates a delay
+between a managed resource being Ready and applying the patch.
+{{< /hint >}}
+
+
+<!-- vale Google.Headings = NO -->
+### CombineFromComposite
+<!-- vale Google.Headings = YES -->
+
+The 
+{{<hover label="combineFromComp" line="12">}}CombineFromComposite{{</hover>}}
+patch takes values from the composite resource, combines them and applies them
+to the managed resource. 
+
+{{< hint "tip" >}}
+Use the 
+{{<hover label="combineFromComp" line="12">}}CombineFromComposite{{</hover>}}
+patch to create complex strings, like security policies and apply them to
+a managed resource. 
+{{< /hint >}}
+
+For example, use the Claim value 
+{{<hover label="combineFromComp" line="15">}}desiredRegion{{</hover>}} and 
+{{<hover label="combineFromComp" line="16">}}field2{{</hover>}} to generate the
+managed resource's
+{{<hover label="combineFromComp" line="20">}}name{{</hover>}}
+
+The 
+{{<hover label="combineFromComp" line="12">}}CombineFromComposite{{</hover>}}
+patch only supports the 
```

Comment:
for a few of these patch types, we say that it only supports X option, then further down after another paragraph we say that it only supports Y strategy.  Should these two sentences be combined into the same paragraph, so the context of "limitations" for this patch type is all grouped together?

e.g., The CombineToComposite patch only supports the `combine` option and the only supported `strategy` is `strategy: string`.

Reply:
Can you give me another example? For this one (and likely the others) I keep them on unique lines to try and make it easier to find the one line/argument/whatever when you're skimming or to be able to understand each line (almost like an API doc) if you're building it for the first time. Open to changing.